releasenotes/notes/27425.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+  - 27425
+releaseNotes:
+  - |
+    **Added** "REPLACE" operation for EnvoyFilter. "REPLACE" opration can replace contents of a named filter with new contents. It is only valid for HTTP_FILTER and NETWORK_FILTER.
```

Comment:
```suggestion
    **Added** "REPLACE" operation for EnvoyFilter. "REPLACE" operation can replace contents of a named filter with new contents. It is only valid for HTTP_FILTER and NETWORK_FILTER. 
```

Reply:
Sorry for the typo, how could I fix it? This has already been merged.